opentreemap/manage_treemap/views/management.py

```diff
@@ -13,15 +14,19 @@
 from django.utils.translation import ugettext as _
```

Comment:
It is my understanding that because `green_infrastructure_labels` is defined at the module level rather than inside a view function using `ugettext` will cause the text to be translated just once when the module is loaded and we would need to use `ugettext_lazy` or move the declaration inside of a view function to ensure that the text is translated on request.